script.js

```diff
@@ -237,3 +237,45 @@ formVal.addEventListener('submit', (e) => {
     errMsgEmail.innerText = 'Your email should not be in Uppercase!';
   }
 });
+
+// PRESERVE DATA IN BROWSER
+
+const Fullnameinput = document.querySelector('.inp1');
+const Firstnameinput = document.querySelector('.inp2');
+const Lastnameinput = document.querySelector('.inp3');
+const Textareainput = document.querySelector('.inp5');
+const Emailinput = emailEl;
+
+const formInfo2 = JSON.parse(localStorage.getItem('formInfo'));
+if (formInfo2) {
+  Firstnameinput.value = formInfo2.Firstname;
+  Fullnameinput.value = formInfo2.Fullname;
+  Lastnameinput.value = formInfo2.Lastname;
+  Textareainput.value = formInfo2.Textarea;
+  Emailinput.value = formInfo2.Email;
+}
+
+formVal.addEventListener('submit', (event) => {
```

Comment:
## Using a submit event handle will not let the data be saved in localStorage as the user is typing his inputs. An input event listener is better suited for this purpose. As required, Kindly ensure the following:

- [ ] After filling the form, and not submitting it, the entered values should be saved in the browser.
- [ ] The contact form information is persistent to page reloads.
- [ ] When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data. 

## EXAMPLE

```
form.addEventListener('input', () => {
  const data = {
    fullName: fullName.value,
    email: email.value,
    message: message.value,
  };

  localStorage.setItem('dataform', JSON.stringify(data));
});
```